composer.json

```diff
@@ -6,24 +6,26 @@
 		"jahuty"
 	],
 	"license": "MIT",
-	"authors": [
-		{
-			"name": "Jack Clayton",
-			"email": "jack@jahuty.com"
-		}
-	],
+  "homepage": "https://www.jahuty.com/",
```

Comment:
das formattinggggg

Reply:
Haha I don't know why Atom is having so much trouble formatting this file. I think I fixed it this time, though.